policyengine_us/parameters/gov/states/nm/usda/snap/min_allotment/amount.yaml

```diff
@@ -0,0 +1,13 @@
+description: New Mexico provides the following monthly SNAP minimum allotment amount.
+values:
+  2024-01-01: 32
+metadata:
+  period: month
+  unit: currency-USD
+  label: New Mexico SNAP minimum allotment amount
+  reference:
+    - title: Income Eligibility Guidelines for SNAP & Financial Assistance
+      href: https://www.hca.nm.gov/wp-content/uploads/FPG-Cards-FY-2024-10.01.23-to-09.30.2024-English-1.pdf
+    # Legal code does not mentioned these values 
+    - title: Report of the Legislative Finance Committee to the Fifty-Sixth Legislature, Second Session
+      href: https://www.nmlegis.gov/Entity/LFC/Documents/Session_Publications/Post_Session_Fiscal_Reviews/2024%20Post%20Session%20web.pdf#page=27
```

Comment:
I don't see this as $32:
>The Income Support Program received a 26.4 percent increase in general fund
revenue. The increase included $14.1 million to increase the Supplemental
Nutrition Assistance Program’s (SNAP) threshold to 200 percent of the federal
poverty level and to increase benefits for clients with higher incomes than SNAP
supports and who are elderly or disabled from $32 monthly to $150 monthly.